cwms-data-api/src/main/java/cwms/cda/data/dao/TimeSeriesDaoImpl.java

```diff
@@ -209,19 +209,12 @@ protected TimeSeries getTimeseries(String page, int pageSize, String names, Stri
         // Now we're going to call the retrieve_ts_out_tab function to get the data and build an
         // internal table from it so we can manipulate it further
         // This code assumes the database timezone is in UTC (per Oracle recommendation)
-        SQL retrieveSelectData = DSL.sql("table(" +
-                CWMS_TS_PACKAGE.call_RETRIEVE_TS_OUT_TAB(
-                        tsId,
-                        unit,
-                        CWMS_UTIL_PACKAGE.call_TO_TIMESTAMP__2(DSL.val(beginTime.toInstant().toEpochMilli())),
-                        CWMS_UTIL_PACKAGE.call_TO_TIMESTAMP__2(DSL.val(endTime.toInstant().toEpochMilli())),
-                        DSL.inline("UTC", String.class),
-                        // All times are sent as UTC to the database, regardless of requested
-                        // timezone.
-                        null, null, null, null, null, null, null,
-                        officeId)
-                + ") retrieveTs"
-        );
+        SQL retrieveSelectData = DSL.sql(
```

Comment:
Its too bad that there isn't a cleaner way to do this in jOOQ.  I hate having to drop into DSL.sql and string concat.

Reply:
Is the idea on this PR that be replacing those literals with binds the DB will then be able to reuse query plans ?  Sounds right - have you measured performance improvements?  How did Dave spot this opportunity?  Did you collect all the CDA queries and have him manually look at them?  Or was he looking at metrics on the database server and noticed the execution plans weren't being reused?  Curious if we can improve our process or automate things

Reply:
That is correct. No specific measurements but it's well documented that not having queries setup with bind really mucks up the query planner and makes even normally fast queries fast which I have seen.

So the actual performance improvement is hard to measure as it's not specifically about this query but the total system.